app/components/efl_qualification_card_component.html.erb

```diff
@@ -1,29 +1,28 @@
-<div class="govuk-grid-column-full">
-  <h2 class="govuk-heading-m govuk-!-margin-top-8" id="english-as-a-foreign-language">English as a foreign language</h2>
-  <p class="govuk-body"><%= qualification_status %></p>
+<h3 class="govuk-heading-m govuk-!-margin-top-6" id="english-as-a-foreign-language">English as a foreign language</h3>
```

Comment:
This should be `h3` as sits within the `QualificationsComponent` which has a `h2` heading